library/globals.inc.php

```diff
@@ -802,6 +802,12 @@
       '2',                              // default = 2
       xl('This will Display the Invoice Number in the Sales Report or the Patient Name and ID or Patient Name and Invoice Number.')
     ), 
+
+    'cash_receipts_report_invoice' => array(
+      xl('Display Invoice Number or Patient Name in the Cash Receipt Report'),
```

Comment:
Cash Receipts Report...forgot an 's'.
I think being able to have both of these would be good.  I have been using both name and invoice in most reports.